shell/components/Resource/Detail/ResourceRow.vue

```diff
@@ -36,13 +39,38 @@ export function extractCounts(labels: string[]): Count[] {
 const {
   label, to, counts, color
 } = defineProps<Props>();
+
+const store = useStore();
+const i18n = useI18n(store);
+
+const displayCounts = computed(() => {
+  if (!counts) {
+    return counts;
+  }
+
+  if (counts.length < 3) {
+    return counts;
+  }
+
+  const [first, ...rest] = counts;
+  const otherCount = sumBy(rest, 'count');
+  const other = {
+    label: i18n.t('generic.other', { count: otherCount }),
+    count: otherCount
+  };
+
+  return [
+    first,
+    other
+  ];
+});
```

Comment:
When rows end up with more than 2 colums we turn it into a 2 column by marking everything else as other.

![image](https://github.com/user-attachments/assets/e5c78306-4f08-4024-8f47-5651a53961d0)